lang/cs/Org.Apache.REEF.IO.TestClient/Org.Apache.REEF.IO.TestClient.csproj

```diff
@@ -0,0 +1,114 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Can you please file a JIRA for it? Otherwise, it is unlikely that we ever act on it :-)